packages/playground/_stories/main/Label/Label.stories.ts

```diff
@@ -37,37 +38,63 @@ const Template: UI5StoryArgs<Label, StoryArgsSlots> = (args) => {
 
 export const Basic = Template.bind({});
 Basic.args = {
-	default: "The quick brown fox jumps over the lazy dog"
+	showColon: true,
 };
+Basic.decorators = [
+	(story, { args }) => {
+		return html`
+	${story({ args: { ...args, for: "myInputSimple", default: "Simple Label"}})}
+	<ui5-input id="myInputSimple"></ui5-input>
```

Comment:
Extract this input into a function and use it in all places. Like the `SetWidth200Px` decorator.